### DISCUSSION TITLE
SEB caused a blue screen in my PC

### DISCUSSION THREAD

**Discussion by luisher29** [SafeExamBrowser/seb-win-refactoring#448] [Q&A]
I tried to exit SEB's red screen to no avail, so I tried restarting my PC. A blank blue screen appeared and I cannot exit it. I tried restarting again, but it won't even turn off. 
![16608455409578135287785494690236](https://user-images.githubusercontent.com/111529630/185463004-525cb73e-ae52-43ab-ba2b-32ded4e3321a.jpg)
Can anybody help me? I was using SEB 3.4.0 on a device with Windows 10.

**Comment by dbuechel** [comment]
What caused the red screen (a.k.a. lock screen) resp. what was the message on it? If you're locked out of your user session, please make sure to press the power button for at least 5 seconds to perform a hard shutdown.
